pkg/cmd/hub/user_proxy.go

```diff
@@ -1,24 +1,23 @@
 package hub
 
 import (
-	"github.com/stolostron/cluster-proxy-addon/pkg/hub"
-
 	"github.com/openshift/library-go/pkg/controller/controllercmd"
 	"github.com/spf13/cobra"
+	"github.com/stolostron/cluster-proxy-addon/pkg/hub"
 	"github.com/stolostron/cluster-proxy-addon/pkg/version"
 )
 
 func NewUserServer() *cobra.Command {
-	userServerOptions := hub.NewUserServerOptions()
+	kubectlUI := hub.NewKubectlUI()
```

Comment:
nit, I would call it HTTPServer or HTTPUserServer. It is not only used by kubectl

Reply:
Name changed.